ebms-payload/src/main/kotlin/no/nav/emottak/payload/ocspstatus/RevocationReason.kt

```diff
@@ -0,0 +1,30 @@
+package no.nav.emottak.payload.ocspstatus
+
+enum class RevocationReason {
+    Unspecified,
+    KeyCompromise,
+    CACompromise,
+    AffiliationChanged,
+    Superseded,
+    CessationOfOperation,
+    CertificateHold;
+
+    companion object {
+        private fun valueOf(i: Int): RevocationReason {
+            values().forEach {
```

Comment:
IntelliJ sier "entries" er recommended. 
```suggestion
            entries.forEach {
```